src/universal-dapp.js

```diff
@@ -537,11 +537,13 @@ UniversalDApp.prototype.runTx = function (args, cb) {
     },
     // run transaction
     function (callback) {
+      var stamp = Date.now()
```

Comment:
Why not `timestamp`?

Reply:
sure i'll update it. Did I even write it?
I actually called it `stamp` everywhere, because I thought you named it `stamp` in "universal-dapp" :P